internal/cmd/logout/command.go

```diff
@@ -55,6 +57,12 @@ func (c *Command) logout(cmd *cobra.Command, _ []string) error {
 			// return err when other than NetrcCredentialsNotFoundErrorMsg or parsing error
 			return err
 		}
+
+		if runtime.GOOS == "darwin" && !c.cfg.IsTest {
```

Comment:
Why don't we delete the credentials for Linux or Windows?

Reply:
This is only for keychain. creds saved in config will be deleted when `persistLogoutConfig` runs

Reply:
that function is called `PersistLogoutToConfig` which handles the authenticated config, but keychain isn't really part of config.

Reply:
Would it make sense to group both forms of persisted credentials into one common function, called `persistLogout()`?